docs/deepnote-mcp.md

```
@@ -0,0 +1,189 @@
+### Connecting Deepnote in ChatGPT UI
+
+While Codex is the recommended OpenAI route when you want the [corresponding plugin for a data-analysis workflow](https://github.com/openai/plugins/tree/main/plugins/deepnote), [Deepnote can also be connected directly in the ChatGPT web app.](https://chatgpt.com/apps/deepnote/asdk_app_69fb51f9519081919c1f3e44ea9a5a05)
+
+1. Open the Deepnote app in ChatGPT.
+2. Select **Sign in with Deepnote**.
+
+![ChatGPT displaying the Deepnote app permission dialog with a Sign in with Deepnote button.](../assets/docs/deepnote-mcp/chatgpt-deepnote-app-permission-dialog.webp)
+
+_Figure 1. Review the Deepnote app permission dialog, then select **Sign in with Deepnote**._
+
+3. Choose the Deepnote workspace ChatGPT may access.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)